cli/command/image/load.go

```diff
@@ -35,12 +40,13 @@ func NewLoadCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags.StringVarP(&opts.input, "input", "i", "", "Read from tar archive file, instead of STDIN")
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress the load output")
+	flags.StringVarP(&opts.name, "name", "n", "", "Name to use when loading OCI image layout tar archive")
```

Comment:
> I assume the oci image layout can have multiple images?

Yes, it does. Please refer to @stevvooe's proposal here https://github.com/docker/docker/issues/25779